Modules/Filtering/ImageFilterBase/wrapping/itkCastImageFilter.wrap

```diff
@@ -15,6 +15,18 @@ itk_wrap_class("itk::CastImageFilter" POINTER_WITH_SUPERCLASS)
         itk_wrap_template("${ITKM_VIF${d}}${ITKM_VID${d}}" "${ITKT_VIF${d}},${ITKT_VID${d}}")
         itk_wrap_template("${ITKM_VID${d}}${ITKM_VIF${d}}" "${ITKT_VID${d}},${ITKT_VIF${d}}")
       endif()
+      foreach(pixel_type "${WRAP_ITK_RGB};${WRAP_ITK_RGBA}")
```

Comment:
This might need to be done only for d=3 (RGB) and d=4 (RGBA).

Reply:
See previous comment chain.